commands/vpc_nat_gateways.go

```diff
@@ -29,7 +29,7 @@ You can use vpc-nat-gateway to perform CRUD operations on a VPC NAT Gateway.`,
 		cmdVPCNATGatewayUpdate,
 	} {
 		AddStringFlag(c, doctl.ArgVPCNATGatewayName, "", "", "Name of the VPC NAT Gateway", requiredOpt())
-		AddStringFlag(c, doctl.ArgVPCNATGatewayType, "", "PUBLIC", "Gateway type", requiredOpt())
+		AddStringFlag(c, doctl.ArgVPCNATGatewayType, "", "PUBLIC", "Gateway type")
```

Comment:
It is a required field on the API but since we are using a default to public since thats the only one supported to the public, i removed the required flag 

Let me know if you think otherwise